### ISSUE TITLE
Better error handle from spawning next build

### ISSUE THREAD

**Issue by khuezy** [serverless-stack/open-next#69]
```typescript
function buildNextjsApp(packager: "npm" | "yarn" | "pnpm") {
  cp.spawnSync(packager, packager === "npm" ? ["run", "build"] : ["build"], {
    stdio: "inherit",
    cwd: appPath,
  });
}
```

open-next should capture any errors and output it to the console.

eg:
```typescript
const spawn =  cp.spawnSync(packager, packager === "npm" ? ["run", "build"] : ["build"], {
    stdio: "inherit",
    cwd: appPath,
  });

if (spawn.stderr) {
  console.log(Error(spawn.stderr))
  process.exitCode = 1;
}
```

**Comment by GinIsTheAnswer** [comment]
I can confirm that the issue is what @edvinerikson pointed out. Manually adding the .cmd to packager solves the build.

It also doesn't spit out spawn.stderr, but rahter spawn.error.

**Comment by fwang** [comment]
Fixed in v0.8.2
